foreman/Dockerfile

```diff
@@ -13,6 +13,12 @@ COPY data_models/dist/data-refinery-models-* data_models/
 RUN pip install \
     data_models/$(ls data_models -1 | sort --version-sort | tail -1)
 
+COPY common/dist/data-refinery-common-* common/
```

Comment:
Hmmm - do you want to keep this as is and copy all of the files or do something like what you have in the `pip install` command above and only copy the newest version?